conf/application.conf

```diff
@@ -197,4 +197,397 @@ versioning {
+                "AN",
+                "AP",
+                "AQ",
+                "AR",
+                "AS",
+                "AT",
+                "AU",
+                "AV",
+                "AW",
+                "AX",
+                "AY",
+                "AZ",
+                "BA",
+                "BB",
+                "BC",
+                "BD",
+                "BE",
+                "BF",
+                "BG",
+                "BH",
+                "BI",
+                "BJ",
+                "BP",
+                "BQ",
+                "BR",
+                "BS",
+                "BT",
+                "BU",
+                "BV",
+                "BW",
+                "BX"
+            ]
+        }
+        "read:individuals-income-laa-c4" {
+            fields = [
+                "A",
+                "B",
+                "C",
+                "D",
+                "E",
+                "F",
+                "G",
+                "J",
+                "AJ",
+                "AP",
+                "BC",
+                "BD",
+                "BE",
+                "BF"]
+        }
+
+        "read:individuals-income-hmcts-c2" {
+            fields = ["H", "J", "K", "M", "N", "O", "R", "T", "U", "AI", "AP", "BC", "BD", "BE", "BF", "BG"]
+        }
+
+        "read:individuals-income-hmcts-c3" {
+            fields = ["H", "J", "K", "M", "N", "O", "R", "T", "U", "AI", "AP", "BC", "BD", "BE", "BF", "BG"]
+        }
+
+        "read:individuals-income-hmcts-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "J", "AJ", "AO", "AP", "BC", "BD", "BE", "BF"]
+        }
+
+        "read:individuals-income-lsani-c1" {
+            fields = ["I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "AA", "AB", "AF", "AH", "AI", "AL", "AM", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "AZ", "BA", "BB", "BC", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BK", "BL", "BM", "BN", "BO", "BP"
+        }
+
+        "read:individuals-income-lsani-c3" {
+            fields = ["I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "AA", "AB", "AF", "AH", "AI", "AL", "AM", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "AZ", "BA", "BB", "BC", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BK", "BL", "BM", "BN", "BO", "BP"
+        }
+
+        "read:individuals-employments-nictsejo-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "N", "O", "P", "Q", "R", "S", "T", "V", "W", "X", "AJ", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "BC", "BD", "BE", "BF", "BH", "BI", "BJ", "BK", "BL", "BM", "BN"]
+        }
+    }
+
+    endpoints {
+        incomeSaSource {
+            endpoint = "/individuals/income/sa/source"
```

Comment:
Update to correct HAL link and ensure endpoint keys match those in the controllers `getEndPointScopes` invocation.

Reply:
The controllers dont currently have this as this is purley the config. We have not added the handler pattern into this repo as of the time of this ticket. These keys can be updated in controllers/config when this gets added in.

Reply:
I have updated the HAL Links however